### DISCUSSION TITLE
i got this error

### DISCUSSION THREAD

**Discussion by WuzenLiz** [fabiocaccamo/django-admin-interface#217] [Q&A]
![image](https://user-images.githubusercontent.com/30171626/204707383-d5277625-9d06-49fd-9b72-7e9998d941fb.png)

**Comment by WuzenLiz** [comment]
need help

**Reply by fabiocaccamo** [reply]
I'm pretty sure the problem is the cache backend you are using, could you share your `CACHES` configuration setting?
